clients/src/main/java/org/apache/kafka/clients/admin/CreateAclsResult.java

```diff
@@ -41,7 +41,7 @@
      * Return a map from ACL bindings to futures which can be used to check the status of the creation of each ACL
      * binding.
      */
-    public Map<AclBinding, KafkaFuture<Void>> results() {
+    public Map<AclBinding, KafkaFuture<Void>> value() {
```

Comment:
I guess it's a matter of perspective. Given the name of the result class, I had assumed the perspective of the map itself being the result value, but I can see what you're saying too. I think the plural does work a little nicer when you use the result directly in a loop (it's a hint that you're working with a collection). Either way works for me.